src/main/kotlin/com/hypto/iam/server/service/OrganizationsService.kt

```diff
@@ -47,41 +44,25 @@ class OrganizationsServiceImpl : KoinComponent, OrganizationsService {
     private val identityProvider: IdentityProvider by inject()
     private val gson: Gson by inject()
     private val txMan: TxMan by inject()
-    private val appConfig: AppConfig by inject()
 
     override suspend fun createOrganization(
-        name: String,
-        description: String,
-        rootUser: RootUser,
-        passcodeStr: String?
+        request: CreateOrganizationRequest
     ): Pair<Organization, TokenResponse> {
-        val secretKey = Constants.SECRET_PREFIX + appConfig.app.secretKey
```

Comment:
Why is this removed ?

Reply:
We will removing all passcodes related to that email and purpose regardless the api is called using the `secretKey`  or a valid `passcode`, so removing this variable as there is no need of it anywhere.